.github/workflows/auto_assign_prs.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Auto Assign PRs'
```

Comment:
this is probably not required. just go with .github/auto_assign.yaml

Reply:
as per the docs here https://github.com/marketplace/actions/auto-assign-action they suggest to have 2 files

Reply:
let me try .. if it works since the name is same .

Reply:
No, it needs separate configuration file for the auto-assign action

Reply:
It is working without the workflow here: https://github.com/openebs/Mayastor/tree/develop/.github

Reply:
right , i was looking for more example like this to better mange the PRs based on the types
 here https://github.com/vmware-tanzu/velero/blob/main/.github/workflows/auto_assign_prs.yml .
 
Should we go with this way or ?

Reply:
I see .. and this gives the flexibility to add auto-labeling job as well.